main.py

```diff
@@ -1,12 +1,31 @@
 from websockets.sync.server import serve
+from websockets.http11 import Response
+from websockets.datastructures import Headers
+import websockets
 import config
 from server import init_server
 import threading
+import json
 
 def main():
-    with serve(init_server.init_server, config.server_ip, 7001) as server:
+    with serve(init_server.init_server, config.server_ip, 7001,
+               process_request=process_request) as server:
         server.serve_forever()
 
+def process_request(connection, headers):
+    path = connection.request.path
+    print(path)
+    if path == "/status":
+        response = bytes(json.dumps({'status': '{}'.format(config.model), 'model': config.model, 'motd': motd()}),'UTF-8')
+        return Response(200, "OK", Headers({'Content-Type': 'application/json', 'Content-Length': len(response)}),
+                        response)
+    return None
+
+def motd():
+    return """[p align=center][font_size=48]OpenLWR[/font_size]
```

Comment:
formatting stuff like this should really be handled on the client

Reply:
my guess is they did it like this so servers can choose what it looks like (though it should be within reason) let me find an example